### ISSUE TITLE
Maybe impose ordering on certain ability categories in the UI

### ISSUE THREAD

**Issue by DoubleMark** [DoubleMark/SynchroDungeonCrawl#3]
I categorized all abilities to kick them out of "Movement", adding our ball spells to "Burnination." This now appears above Movement. I feel like some categories (like movement and basic attacks) may want to appear first in the list for consistency across characters. Thoughts?

**Comment by cesium12** [comment]
Yeah, that'd be great, huh? Wanna come up with some sort of extensible scheme for specifying category order? I'm not sure at all what that should look like.
